src/sentry/runner/commands/run.py

```diff
@@ -528,11 +530,30 @@ def ingest_consumer(consumer_types, all_consumer_types, **options):
 
 @run.command("ingest-metrics-consumer")
 @log_options()
-@click.option("--topic", default="ingest-metrics", help="Topic to get subscription updates from.")
+@click.option("--topic", default="ingest-metrics", help="Topic to get metrics data from.")
 @batching_kafka_options("ingest-metrics-consumer")
 @configuration
 def metrics_consumer(**options):
 
     from sentry.sentry_metrics.indexer.indexer_consumer import get_metrics_consumer
 
     get_metrics_consumer(**options).run()
+
+
+@run.command("ingest-metrics-consumer-2")
+@log_options()
+@click.option("--topic", default="ingest-metrics", help="Topic to get metrics data from.")
+@batching_kafka_options("ingest-metrics-consumer")
+@configuration
+@click.option(
+    "--processes",
+    default=1,
```

Comment:
When you start this in prod. Let's start with 4. 
We have metrics to see their utilization: https://github.com/getsentry/arroyo/blob/main/arroyo/processing/strategies/streaming/transform.py#L321

Then if they are saturated (hopefully not) raise that number.
Then gradually scale down the replicas (that will increase the multi process usage) till we can.